hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/selective/BalancingJob.java

```diff
@@ -44,10 +46,17 @@ public BalancingJob(ConsumerNodesRegistry consumersRegistry,
         this.workTracker = workTracker;
         this.metrics = metrics;
         this.kafkaCluster = kafkaCluster;
-        this.executorService = Executors.newSingleThreadScheduledExecutor();
+        ThreadFactory threadFactory = new ThreadFactoryBuilder()
+                .setNameFormat("BalancingExecutor-%d").build();
+        this.executorService = Executors.newSingleThreadScheduledExecutor(threadFactory);
         this.intervalSeconds = intervalSeconds;
 
         metrics.registerGauge(
+                "consumers-workload." + kafkaCluster + ".selective.all-assignments",
```

Comment:
\[PMD\] INFO: The String literal "consumers\-workload\." appears 4 times in this file; the first occurrence is on line 55